root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Framework/Js/common.js

```diff
@@ -0,0 +1,623 @@
+﻿//**********************************************************************************
+//* All Rights Reserved, Copyright (C) 2007,2012 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+// ------------------------------------------------------------
+//  イベントハンドラ
+// ------------------------------------------------------------
+
+// ---------------------------------------------------------------
+// ページがロードされた時に呼ばれる
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2015/02/06  Supragyan         Added condition for check AjaxPostBackElement in Fx_AjaxExtensionInitializeRequest
+//*  2015/02/06  Supragyan         Added condition for check AjaxPostBackElement in Fx_AjaxExtensionEndRequest
+//*  2015/02/09  Supragyan         Added condition for Trident on Internet Explorer
+//*  2015/09/09  Sandeep           Added condition code to detect IE-9, IE-10 and IE-11, to suppress double transmission
+//*  2015/12/28  Sai               Added Java script method for preventing session timeout.
+//*  2015/12/28  Sai               Commented out window.setInterval method.
+//*  2016/01/11  Sai               Removed unnecessary code.
+//*  2016/01/12  Sai               Changed interval in method window.setInterval(HttpPing, 5000)
+//*  2016/01/13  Sai               Removed Ajax extensions code and added JQuery's Ajax Complete method.
+//*  2016/01/22  Sai               Added .ajaxSend method to prevent double submit functionality in Ajax form.
+//*  2016/01/22  Sai               Added flag variable 'PreventAjaxDoubleSubmit', added if condiation to this flag in 
+//*                                ajaxSend method to skip progress dialogue also added codign for setting this flag to 
+//*                                false in ajaxComplete method.   
+//*  2016/02/01  Sai               Fixed Progress dialogue mask not displaying problem.
+//**********************************************************************************
+
+function Fx_Document_OnLoad() {
+    // OnLoad処理を別スレッドで実行
+    setTimeout("Fx_Document_OnLoad2()", 1);
+}
+
+// ---------------------------------------------------------------
+// ページがロードされた時に呼ばれる
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+function Fx_Document_OnLoad2() {
+    window.returnValue = "";
+
+    // マスクの初期化
+    Fx_CreateMask();
+
+    // プログレス ダイアログの初期化
+    Fx_InitProgressDialog();
+
+    // Webサーバへ一定時間ごとにpingを行う
+   //window.setInterval(HttpPing, 5 * 60 * 1000);
```

Comment:
@yusukemaegawa 

This explanation is not exist.
How about you describe this to pukiwiki ?